vassal-app/src/main/resources/VASSAL/i18n/Editor.properties.orig

```diff
@@ -0,0 +1,551 @@
+# Module Editor Properties
```

Comment:
This file shouldn't have been committed?

Reply:
Merge tool detritus